open_spiel/algorithms/dqn_torch/dqn.cc

```diff
@@ -141,37 +142,37 @@ void DQN::AddTransition(const State& prev_state,
 }
 
 Action DQN::EpsilonGreedy(std::vector<float> info_state,
-                          std::vector<Action> legal_actions,
-                          double epsilon) {
+                          std::vector<Action> legal_actions, double epsilon) {
   Action action;
   if (legal_actions.empty()) {
     // In some simultaneous games, some players can have no legal actions.
-    return 0;
```

Comment:
This should be zero actually (see other comment)

Reply:
Done